### ISSUE TITLE
apple_bundle_version for ios_static_library

### ISSUE THREAD

**Issue by armcknight** [bazelbuild/rules_apple#735]
Hello, I noticed that there is a parameter in both ios_framework and ios_static_library for apple_bundle_version. I see when I build an ios_framework that includes such, that it inserts the version number into the framework bundle's Info.plist. However, I can't find anywhere that does something equivalent for ios_static_library. To my knowledge there is no other way to pass a version in a static lib than to hardcode it in source.

**Comment by thii** [comment]
Static frameworks don’t have to be versioned, and don’t have an Info.plist file (they don’t need one).

**Comment by armcknight** [comment]
Exactly, I’m wondering if this option shouldn’t be removed from it? It seems like a red herring.

**Comment by thii** [comment]
Static libraries/frameworks will be staticlly linked to the app binary anyway, what do you need the version for?

That being said, you can still add your own Info.plist to the final framework using ipa_post_processor.

**Comment by armcknight** [comment]
The original reason I wanted to track the version is for some internal purposes, sending it along with some network requests from my library. I’ve solved that problem in other ways.

The reason I opened this issue is to see what the project maintainers want for this parameter, because as far as I can tell it’s a no-op for ios_static_library and should be removed.
